lib/Commands/BuildAllCommand.php

```diff
@@ -43,6 +45,20 @@ protected function configure(): void
                 InputOption::VALUE_NONE,
                 'Clear the build cache before building everything.',
             )
+            ->addOption(
+                'project',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'The project to build the docs for.',
+                '',
+            )
+            ->addOption(
+                'libversion',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'The project version to build the docs for. Only usable in combination with --project option',
```

Comment:
Would it make sense to have a notation instead like `--project orm@version` or `orm:version`?